Jenkinsfile-Release

```diff
@@ -56,6 +56,11 @@ dockerizedBuildPipeline(
 
     jira.versionPrefix = jiraVersionMappings[chart]
     jiraSetFixVersion(jira, version)
+
+    verifyDownloadLinks(
```

Comment:
I think this is not going to work, as it is going to take some time for `artifacthub` to check the repo and update the chart version. It is going to take 30 to 60 mins.

I think instead we may verify the chart binary is uploaded. This an example URL `https://sonatype.github.io/helm3-charts/nexus-iq-server-132.0.2.tgz`

```
verifyDownloadLinks(
        urlParts: ['https://sonatype.github.io/helm3-charts/', chart, '-', version],
        urlSuffixes: ['.tgz']
    )
```

Reply:
Thanks. I wasn't aware of that. I applied your suggestion in [c867194](https://github.com/sonatype/helm3-charts/pull/173/commits/c86719466e40ffcfae7fc258e2b5d6a6d105b120)